### ISSUE TITLE
Metric timestamp

### ISSUE THREAD

**Issue by lboudard** [logstash-plugins/logstash-input-cloudwatch#23]
Hi,

I'm trying to use this plugin to import custom metrics. Though, I noticed that metric's stats timestamp was overriden by end_time of the requested period, which sounds inconsistent to me.
```
  def cleanup(event)
    event.delete :statistics
    event.delete :dimensions
    event[:start_time] = Time.parse(event[:start_time]).utc
    event[:end_time]   = Time.parse(event[:end_time]).utc
    event[:timestamp]  = event[:end_time]
    LogStash::Util.stringify_symbols(event)
  end
```
Any reason for this behavior?
Thanks!

**Comment by rahulsinghai** [comment]
Following line should be removed:
```
event[:timestamp]  = event[:end_time]
```

**Comment by Jeck-96** [comment]
Hi, 
I agree with you, I noticed the same thing. 
In the configuration file of logstash to import data from cloudwatch, I choose an interval of 15 minutes and a period of 5 minutes. So into logstash it upload two document, each document rapresent data collected in a period(5 minutes). So documents in the same interval should have a different timestamp, spaced five minutes, but they have got the same timestamp.

**Comment by Jeck-96** [comment]
Hi,
I agree with you, I noticed the same thing.
In the configuration file, I choosed an interval of 15 minutes and a period of 5 minutes. So into logstash it upload two documents, each document represents data collected in a period (5 minutes). Documents in the same interval should have a different timestamp, spaced five minutes, but they have got the same timestamp. 
It's possible solve this issue?

**Comment by Jeck-96** [comment]
Hi, 

I agree with you, I noticed the same thing.
In the configuration file to import data from cloudwatch to elasticsearch I choosed an interval of 15 minutes and a period of 5 minutes. In elasticsearch it upload two document foreach interval, each document refers data of a period. So each document shuld be spaced five minutes, but in the same interval, they have got the same timestamp field.
It's possible solve the issue?
